app/models/item.rb

```diff
@@ -7,10 +8,16 @@ class Item <ApplicationRecord
   validates_presence_of :name,
                         :description,
                         :price,
-                        :image,
                         :inventory
   validates_inclusion_of :active?, :in => [true, false]
   validates_numericality_of :price, greater_than: 0
+  validates_numericality_of :inventory, greater_than: 0
+  
+  def set_defaults
```

Comment:
I like the way default images are set here

Reply:
yeah we tried to set them in the migration but rails did not really like that